workflows/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'backend'
```

Comment:
This is not the root project - please remove settings.gradle.

Reply:
Done